checkbox-ng/plainbox/impl/session/test_remote_assistant.py

```diff
@@ -46,6 +58,215 @@ def not_allowed(self, *args):
         with self.assertRaises(AssertionError):
             not_allowed(self_mock)
 
+    @mock.patch("plainbox.impl.secure.sudo_broker.is_passwordless_sudo")
+    def test__reset_sa(self, is_passwordless_sudo_mock):
+        with mock.patch.object(SessionAssistant, "__init__") as init_mock:
```

Comment:
Why do we do it this way in this function? Can't we mock above the function the SessionAssistant and check it was called?

Reply:
We can. Changed for consistency. Thanks.